projects/hello_world/src/main.c

```diff
@@ -0,0 +1,97 @@
+#include "../../../libraries/ms-common/inc/adc.h"

```

Comment:
Hmm this is strange, you should be able to just do `#include "adc.h"`

Reply:
my vscode had an interesting file also called adc.h so i had to include the direct path